File: IT/1/Strutture dati/Esercizi/Python/esercizio15.py
# file esercizio15.py

from __future__ import print_function
from random import randint

nums = [randint(1, 100) for n in range(10)]

print(nums)

nums.sort(reverse=True)

print(nums)

maxIndex = 0
print("Numero maggiore in lista:", nums[maxIndex])

minIndex = len(nums)-1
print("Numero minore in lista:", nums[minIndex])

File: IT/1/Strutture dati/Esercizi/Java/Esercizio15.java
/* file Esercizio15.java */

import java.util.Arrays;
import java.util.Collections;
import java.util.Random;

public class Esercizio15 {
	public static void main(String [] args) {
		Random rand = new Random();
		// dobbiamo usare il tipo "Integer" per utilizzare i metodi della classe Collection
		Integer [] nums = new Integer[10];

		for (int i = 0; i < nums.length; i++) {
			int r = rand.nextInt(100) + 1;
			nums[i] = r;
		}

		System.out.println(Arrays.toString(nums));

		Arrays.sort(nums, Collections.reverseOrder());

		System.out.println(Arrays.toString(nums));

		int maxIndex = 0;
		System.out.println("Numero maggiore nell'array: " + nums[maxIndex]);

		int minIndex = nums.length - 1;
		System.out.println("Numero minore nell'array: " + nums[minIndex]);
	}
}

